exercises/poker/src/test/java/PokerTest.java

```diff
@@ -0,0 +1,164 @@
+import static org.junit.Assert.assertEquals;
+
+import java.util.Arrays;
+
+import org.junit.Test;
+
+import jdk.nashorn.internal.ir.annotations.Ignore;
+
+public class PokerTest {
+    @Test
+    public void oneHand() {
+        final String hand = "4S 5S 7H 8D JC";
+        assertEquals(Arrays.asList(hand),new Poker(Arrays.asList(hand)).getBestHands());
+    }
+
+    @Test
+    @Ignore
```

Comment:
Sorry, I just noticied one more thing from the [POLICIES doc](https://github.com/exercism/java/blob/master/POLICIES.md#ignore-noninitial-tests). The `@Ignore`s should have a hint: `@Ignore("Remove to run test")`. This is to make it more clear to the users that they have to remove that line to get the test to run :)